### DISCUSSION TITLE
Custom font& Linking to Another Page?

### DISCUSSION THREAD

**Discussion by softwaredevkev** [TriliumNext/Trilium#1615] [Q&A]
Is there a way to have a custom font besides just bold, italics, headers, and coloring? 

Also, is there a way to link to another page/note? For instance, if I have some text in one note, is there a way to highlight that text and have it link to another page?

**Comment by zadam** [comment]
No, custom fonts are not supported.

Linking, yes, that's one of the core features. Either press `CTRL-L` or select the text and:

![image](https://user-images.githubusercontent.com/617641/107149303-28f33c80-6958-11eb-9d37-bb6ddeba46bb.png)
